header/header.go

```diff
@@ -82,6 +82,16 @@ func (eh *ExtendedHeader) LastHeader() bts.HexBytes {
 	return eh.RawHeader.LastBlockID.Hash
 }
 
+// IsBefore returns whether the given header is of a higher height.
+func (eh *ExtendedHeader) IsBefore(h *ExtendedHeader) bool {
```

Comment:
Shouldn't this check include `==` as well? Or at least, on line 258, do 

`if !sbjHead.IsBefore && !sbjHead.Equals=`

Reply:
Nope

Reply:
Ignore the above ^ 

the diff has changed since